### DISCUSSION TITLE
Disabling IME input in WPF cef browser

### DISCUSSION THREAD

**Discussion by simpleDev5** [cefsharp/CefSharp#4805] [General]
I am making a visual studio extension which displays a webpage on a tool window using the CefBrowser.
I am using WPF browser to display the webpage. 
When I change the input language to Korean , the IME pop-up shows up.
Is there a way to disable the IME popup in Wpf?
![cefIME](https://github.com/cefsharp/CefSharp/assets/170286746/0583cb23-8bee-4b2c-a29c-a566b5a5152d)

I tried with the minimal example and I could see that the ime popup doesn't show up for winforms browser.
Is there a way to achieve the same behavior using WPF browser? So that the experience is seamless

**Comment by amaitland** [comment]
> Is there a way to disable the IME popup in Wpf?

You can enable the IME keyboard handler as per https://github.com/cefsharp/CefSharp/issues/1262#issuecomment-1084007691

Which should give a better experience.

No idea how to **disable IME**. I don't use IME personally.

> I tried with the minimal example and I could see that the ime popup doesn't show up for winforms browser.

The equivalent of using the `WinForms` version in `WPF` is available at https://github.com/cefsharp/CefSharp.Wpf.HwndHost
